spec/company-detail/ai-company-summary.md

```
@@ -0,0 +1,151 @@
+
+### 2. 뉴스 제목
+
+- Google News RSS에서 가져온 최근 20~30개 뉴스 제목
+
+### 3. 기업 기본정보
+
+- 회사명, 업종
+
+## 데이터 흐름
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)